DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/CourseDelegatesControllerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.TrackingSystem.Delegates
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.CourseDelegates;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using FakeItEasy;
+    using FluentAssertions.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class CourseDelegatesControllerTests

```

Comment:
Could also test:
* AllCourseDelegates gets courses for the user's centre only.
* Index intreprets the user's category id correctly

Reply:
Since the refactor to make the dataservices compare against a null category id rather than 0, the user category interpretation is taken care of in the new CourseCategoryHelper with its own tests there.